dask_jobqueue/pbs.py

```diff
@@ -1,42 +1,44 @@
 import logging
 import os
-import socket
-import sys
-
-from distributed import LocalCluster
-from distributed.utils import get_ip_interface
 
 from .core import JobQueueCluster
 
 logger = logging.getLogger(__name__)
 
-dirname = os.path.dirname(sys.executable)
-
 
 class PBSCluster(JobQueueCluster):
     """ Launch Dask on a PBS cluster
 
     Parameters
     ----------
     name : str
-        Name of worker jobs. Passed to `$PBS -N` option.
+        Name of worker jobs and Dask workers. Passed to `$PBS -N` option.
     queue : str
         Destination queue for each worker job. Passed to `#PBS -q` option.
     project : str
         Accounting string associated with each worker job. Passed to
         `#PBS -A` option.
-    threads_per_worker : int
+    resource_spec : str
+        Request resources and specify job placement. Passed to `#PBS -l`
+        option.
+    walltime : str
+        Walltime for each worker job.
+    job_extra : list
+        List of other PBS options, for example -j oe. Passed with '#PBS ' prefix
+    local_directory : str
+        Dask worker local directory for file spilling.
```

Comment:
Why remove threads_per_worker above?  Also did you want to logically separate the dask options from the PBS options?

Reply:
Ah, scrolling down I see that you've done just that.  If so then local_directory should probably be an inherited parameter.  I *think* it should be generic.

Reply:
Yup I separated options, and for more consistancy with other keywords I used `threads` instead of `thread_per_worker`. 
`local_directory` is generic, but I wanted to override it in PBSCluster, because standard PBS installation always provide a $TMPDIR variable which resolve to something like /tmp/{JOB_ID}/. I wanted to make this as the default local_directory for PBSCluster workers.